pkg/oc/lib/newapp/app/pipeline.go

```diff
@@ -379,13 +380,13 @@ func AddServices(objects Objects, firstPortOnly bool) Objects {
 	svcs := []runtime.Object{}
 	for _, o := range objects {
 		switch t := o.(type) {
-		case *appsapi.DeploymentConfig:
-			svc := addServiceInternal(t.Spec.Template.Spec.Containers, t.ObjectMeta, t.Spec.Selector, firstPortOnly)
+		case *appsv1.DeploymentConfig:
+			svc := addService(t.Spec.Template.Spec.Containers, t.ObjectMeta, t.Spec.Selector, firstPortOnly)
 			if svc != nil {
 				svcs = append(svcs, svc)
 			}
-		case *extensions.DaemonSet:
-			svc := addServiceInternal(t.Spec.Template.Spec.Containers, t.ObjectMeta, t.Spec.Template.Labels, firstPortOnly)
+		case *kappsv1.DaemonSet:
```

Comment:
You might need more than just that: `*extensionsv1beta1.DaemonSet, *kappsv1beta2.DaemonSet, *kappsv1.DaemonSet` taken from https://github.com/openshift/origin/blob/master/pkg/oc/cli/set/triggers.go

Reply:
Thanks, updated to take other groupversions into account